db/migrate/20170608155514_add_training_session_id_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTrainingSessionIdToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :training_session_id, :string
```

Comment:
Oh yeah this makes sense. `training_session_id` seems unnecessary since it's only the last training session the user was in.